packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -218,7 +218,11 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
         )
       }
     } else {
-      await this._schemaValidator.validateSchema(content, model.content.schema)
+      await this._schemaValidator.validateSchema(
+        content,
+        model.content.schema,
+        model.commitId.toString()
+      )
```

Comment:
Maybe just pass the content and the model to `validateSchema(...)` and let it handle cache keys inside?

Reply:
Hmm, good suggestion!

Reply:
Oh, sorry, no. I can not generalise it for all the StreamUtils :/ There is no Model for Tile.